From 42c3e811c5aa46e16b92cb0ba52c1051676294c0 Mon Sep 17 00:00:00 2001
From: Naveen Raj Ranganathan <naver@qti.qualcomm.com>
Date: Thu, 28 May 2026 16:41:49 +0530
Subject: [PATCH] init/main.c: export saved_command_line variable

saved_command_line is not visible to out of source tree modules, this changes makes it visible to loadable kernel modules.

Change-Id: Ic43200ec7ce172b843e45f8481f6ceb56d9bef01
Signed-off-by: Naveen Raj Ranganathan <naver@qti.qualcomm.com>
---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index bdc40307e2a0..af84b04e24a7 100644
--- a/init/main.c
+++ b/init/main.c
@@ -140,6 +140,7 @@ char __initdata boot_command_line[COMMAND_LINE_SIZE];
 /* Untouched saved command line (eg. for /proc) */
 char *saved_command_line __ro_after_init;
 unsigned int saved_command_line_len __ro_after_init;
+EXPORT_SYMBOL(saved_command_line);
 /* Command line for parameter parsing */
 static char *static_command_line;
 /* Untouched extra command line */
-- 
2.34.1

